lib/gemstash/migrations/02_authorizations.rb

```diff
@@ -2,7 +2,7 @@
   change do
     create_table :authorizations do
       primary_key :id
-      String :auth_key, :size => 2056, :null => false
+      String :auth_key, :size => 255, :null => false
```

Comment:
I believe the auth key can be user provided. The generated value is used if the user doesn't explicitly give a value. Should this limit be indicated in the docs perhaps? Is it possible there will come a day when 255 won't be enough for a secure auth token? Should we consider using a data type that works with the larger size with MySQL rather than reduce the size? Or should we document the limit and just migrate the table later if we want to expand it?